base/applications/sndvol32/sndvol32.c

```diff
@@ -1211,6 +1212,21 @@ MainWindowProc(HWND hwnd,
                 /* Disable the 'Advanced Controls' menu item */
                 EnableMenuItem(GetMenu(hwnd), IDM_ADVANCED_CONTROLS, MF_BYCOMMAND | MF_GRAYED);
 
+                /* Load the placement coordinate data of the window */
+                bRet = LoadXYCoordWnd(Preferences);
+                if (!bRet)
+                {
+                    /*
+                     * LoadXYCoordWnd() may fail for the first time when the Sound Volume applet is launched.
+                     * This is normal because the registry settings weren't created for the applet at that point.
+                     */
+                    MixerWindow->WndPosX = 0;
+                    MixerWindow->WndPosY = 0;
```

Comment:
Should you just skip the SetWindowPos call in this case? Moving to the top left corner seems odd.